wormhole-connect/src/views/v2/Bridge/Routes/SingleRoute.tsx

```diff
@@ -0,0 +1,318 @@
+  },
+  card: {
+    width: '100%',
+    cursor: 'pointer',
+    maxWidth: '420px',
+  },
+}));
+
+type Props = {
+  config: RouteData;
+  available: boolean;
+  isSelected: boolean;
+  showDestinationGasFee?: boolean;
+  title?: string;
+  onSelect?: (route: Route) => void;
+};
+
+const SingleRoute = (props: Props) => {
+  const { classes } = useStyles();
+  const theme = useTheme();
+
+  const {
+    fromChain: sourceChain,
+    token: sourceToken,
+    toChain: destChain,
+    destToken,
+    amount,
+  } = useSelector((state: RootState) => state.transferInput);
+
+  const { toNativeToken } = useSelector((state: RootState) => state.relay);
+
+  const [isFeesBreakdownOpen, setIsFeesBreakdownOpen] = useState(false);
+
+  const { name, route, providedBy } = props.config;
+
+  // Compute the fees for this route
+  const {
+    receiveAmount,
+    receiveAmountUSD,
+    estimatedTime,
+    isFetching: isFetchingFees,
+  } = useComputeFees({
+    sourceChain,
+    destChain,
+    sourceToken,
+    destToken,
+    amount,
+    route,
+    toNativeToken,
+  });
+
+  // Compute the quotes for this route
+  const {
+    receiveNativeAmt,
+    relayerFee,
+    isFetching: isFetchingQuote,
+  } = useComputeQuoteV2({
+    sourceChain,
+    destChain,
+    sourceToken,
+    destToken,
+    amount,
+    route,
+    toNativeToken,
+  });
+
+  const totalFees = useMemo(() => {
+    let total = 0;
+
+    if (relayerFee) {
+      total += relayerFee || 0;
+    }
+
+    if (props.showDestinationGasFee && receiveNativeAmt) {
+      total += receiveNativeAmt || 0;
+    }
+
+    return (
+      <>
+        <Typography color={theme.palette.text.secondary} fontSize={14}>
+          Total fees
+        </Typography>
+        {isFetchingFees || isFetchingQuote ? (
+          <CircularProgress size={14} />
+        ) : (
+          <Typography fontSize={14}>
+            {Number.parseFloat(toFixedDecimals(`${total}`, 6))}
+          </Typography>
+        )}
+      </>
+    );
+  }, [
+    isFetchingFees,
+    isFetchingQuote,
+    props.showDestinationGasFee,
+    relayerFee,
+    receiveNativeAmt,
+  ]);
+
+  const bridgeFee = useMemo(
+    () => (
+      <Stack direction="row" justifyContent="space-between">
+        <Typography
+          color={theme.palette.text.secondary}
+          fontSize={14}
+          paddingLeft="8px"
+        >
+          Bridge fee
+        </Typography>
+        {isFetchingQuote ? (
+          <CircularProgress size={14} />
+        ) : (
+          <Typography fontSize={14}>{relayerFee}</Typography>
+        )}
+      </Stack>
+    ),
+    [isFetchingQuote, relayerFee],
+  );
+
+  const destinationGasFee = useMemo(() => {
+    if (!receiveNativeAmt) {
+      return null;
+    }
+
+    return (
+      <Stack direction="row" justifyContent="space-between">
+        <Typography
+          color={theme.palette.text.secondary}
+          fontSize={14}
+          paddingLeft="8px"
+        >
+          Destination gas fee
```

Comment:
Same comment here. This isn't a fee you are paying, it's an amount you are receiving.

Reply:
Same response as above.